Sources/SwiftDocC/Model/Rendering/References/TopicColor.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+/// A color that associated with a documentation topic.
+public struct TopicColor: Codable, Hashable {
```

Comment:
Maybe this should be a protocol and standard colors can then be an implementation of the protocol?